src/plugins/admonition/schema.js

```diff
@@ -14,13 +12,6 @@ function normalizeAdmonition(change, error) {
     switch (code) {
     // Admonition type is not valid.
     case 'node_data_invalid':
-        if (key === 'class') {
```

Comment:
Why should we omit this fix? I know that this will not happen in our application because of front end validation, but shouldn't `cnx-designer` be standalone package?

Reply:
nvm, there is separate Plugin for this :D

Reply:
I've extracted classes to a separate plugin as it limits code duplication and makes it easier to add support for new elements and to test it (since we now only need to test it once, and not once per element).